Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -24,6 +24,12 @@ open class NavigationEventsManager {
     
     weak var dataSource: EventsManagerDataSource?
     
+    /**
+     Optional app metadata that can be used to associate app state, specifically app name and version, with events in the telemetry pipeline.
+     Specifically, developers can provide application name and version, user identifier, and session identifier.
```

Comment:
The developer is the audience, so we can address them directly using the second person. It’s also a bit awkward to talk about the telemetry “pipeline”, since the telemetry backend isn’t accessible to the developer.

```suggestion
     Optional application metadata that that can help Mapbox more reliably diagnose problems that occur in the SDK.
     
     For example, you can provide your application’s name and version, a unique identifier for the end user, and a session identifier.
```

Reply:
I'll make this change.